content/10-minute-tutorial.md

```diff
@@ -15,8 +15,44 @@ To get started with Cucumber in Java, you will need the following:
 
 - A Cucumber plugin for your IDE, for example [IntelliJ IDEA Cucumber for Java plugin](https://plugins.jetbrains.com/plugin/7212-cucumber-for-java) to go with IntelliJ IDEA
 
+{{% /block %}}
+
+{{% block "javascript" %}}
+
+To get started with Cucumber in JavaScript, you will need the following:
+
+- node.js (v4 and up)
+
+- npm
+
+**Install node.js**
+
+On Mac OS, you can install `node.js` using homebrew: `brew install node`.
```

Comment:
I'd also suggest something like: https://github.com/tj/n

Reply:
Not familiar with this myself, so unless you specifically tell me what to write / add it yourself..?